openfisca_core/commons/tests/test_formulas.py

```diff
@@ -0,0 +1,55 @@
+import datetime
```

Comment:
@benjello Would you please take a look at this test and tell me if it describes the intended `switch` "contract" (input/output types).

Reply:
Link does not work ...

Reply:
https://github.com/openfisca/openfisca-core/blob/27d463959d91f590728aa594329ed353f8f140e4/openfisca_core/commons/tests/test_formulas.py

Reply:
I guess so.